### PR TITLE
Use benchmark-data git repo for storing data

### PR DESCRIPTION
**Pull request by isuruf**

### CODE REVIEW

**Comment by isuruf** [comment]
cc @inducer (you might not be watching this repo, hence the ping)

File: build-and-benchmark-py-project.sh [hunk 1]
```diff
@@ -58,6 +59,16 @@ if [[ "$output" = *"worse"* ]]; then
...
+  export GIT_AUTHOR_NAME="Automated Benchmark Bot"
+  export GIT_AUTHOR_EMAIL="bot@gitlab.tiker.net"
+  export GIT_COMMITTER_NAME=$GIT_AUTHOR_NAME
+  export GIT_COMMITTER_EMAIL=$GIT_AUTHOR_EMAIL
+  git commit -m "Update benchmark data for $main_commit" --allow-empty
+  ssh-agent bash -c "ssh-add $PWD/../.deploy_key; git push"
+  cd ..
   git branch -v
+  mkdir -p ~/.scicomp-benchmarks/asv
   asv publish --html-dir ~/.scicomp-benchmarks/asv/$PROJECT
```
Comment (inducer) [inline build-and-benchmark-py-project.sh]: I'm not a huge fan of the fact that this is not local to the build. Any chance we could move this?

Comment (isuruf) [reply build-and-benchmark-py-project.sh]: Fixed it

**Comment by inducer** [comment]
Thanks!

**Pull request merged**